R/utils.R

```diff
@@ -26,9 +26,13 @@ formula.xbal<-function(x,...){
 ##' @return Result of \code{fun}.
 withOptions <- function(optionsToChange, fun) {
   oldOpts <- options()
-  on.exit(options(oldOpts))
   options(optionsToChange)
-  tryCatch(fun(), finally = options(oldOpts))
+  # store the old values of the options, just for the options that were changed
+  old.opt.values <- list()
```

Comment:
1. It seems that if this works, then  
```r
old.opt.values <- oldOpts[names(optionsToChange)]
```
should also work as a replacement for the for loop, and be a bit easier to read.  Does it?

2. I'd suggest maintaining the naming conventions of the RItools codebase, which from the surrounding code would appear to favor `lowerCamelCase` over `period.separated`.

Reply:
(I'm marking this as resolved, given my comment/commit below incorporating these suggestions)